easybuild/easyconfigs/m/Mercurial/Mercurial-5.7.1-foss-2020b.eb

```diff
@@ -0,0 +1,33 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = "PythonPackage"
+
+name = 'Mercurial'
+version = '5.7.1'
+
+homepage = 'http://mercurial.selenic.com/'
+description = """Mercurial is a free, distributed source control management tool. It efficiently handles projects
+of any size and offers an easy and intuitive interface.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
```

Comment:
I think this should be moved to `GCCcore`. Note that that will mean adding `builddependencies = [('binutils', '2.35')]`

Reply:
Ok, thank you. I was thinking about it as this would follow the new approach but since the previous version, 3.6.x was with foss, I have followed the same approach. I will do it.